pkgs/development/python-modules/rfeed/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, lxml
+}:
+
+buildPythonPackage rec {
+  pname = "rfeed";
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-qpUG8oZrdPWjItOUoUpjwZpoJcLZR1X/GdRt0eJDSBk=";
+  };
+
```

```suggestion

  pythonImportsCheck = [ "rfeed" ];
```
refer, https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck